addons/base_import/models/base_import.py

```diff
@@ -1,25 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
 import csv
 import datetime
 import io
 import itertools
 import logging
+import psycopg2
 import operator
 import os
 import re
 
-from openerp.tools.mimetypes import guess_mimetype
+from odoo import api, fields, models
+from odoo.tools.translate import _
+from odoo.tools.mimetypes import guess_mimetype
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT, DEFAULT_SERVER_DATETIME_FORMAT
 
 try:
     from cStringIO import StringIO
 except ImportError:
     from StringIO import StringIO
 
-import psycopg2
-
-from openerp.osv import orm, fields
-from openerp.tools.translate import _
-from openerp.tools import DEFAULT_SERVER_DATE_FORMAT, \
-    DEFAULT_SERVER_DATETIME_FORMAT
```

Comment:
The import order change is incorrect, StringIO/cStringIO are part of the stdlib and psycopg2 isn't